contexts/StakingContext.tsx

```diff
@@ -9,7 +9,7 @@ import {
 } from "hooks";
 import { createContext, ReactNode } from "react";
 
-export interface BalancesContextState {
+export interface StakingContextState {
```

Comment:
I renamed this context because it also contains information about delegation, so I felt this name is more appropriate